data/src/main/java/com/onewelcome/data/di/DataStoreModule.kt

```diff
@@ -0,0 +1,27 @@
+package com.onewelcome.data.di
+
+import android.content.Context
+import androidx.datastore.core.DataStore
+import androidx.datastore.preferences.core.PreferenceDataStoreFactory
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.preferencesDataStoreFile
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object DataStoreModule {
+  private const val DATA_STORE_PREFERENCES = "data_store_preferences"
```

Comment:
I think this could be renamed to something like `user_preferences_data_store` or `user_profile_data_store` to better describe purpose of this data store

Reply:
Well we do not know yet what will the purpose of this data store 😅 It's just a dummy file with example implementation, it can be renamed as fit later on.